models/evaluator.py

```diff
@@ -0,0 +1,27 @@
+from nltk.translate.bleu_score import sentence_bleu
+
+
+def evaluation(translations, references):
```

Comment:
what is the name of this evaluation, you can use its name here, for example, `bleu_evaluation` not only simple `evaluation` naming